doc/src/guides/solving/solve-polynomial.md

```diff
@@ -0,0 +1,514 @@
+        / 5           \ 
+[CRootOf\x  - x + 1, 0/]
+>>> Poly(fifth_order, x).all_roots()
+        / 5           \         / 5           \         / 5           \       
+[CRootOf\x  - x + 1, 0/, CRootOf\x  - x + 1, 1/, CRootOf\x  - x + 1, 2/, CRoot
+<BLANKLINE>
+    / 5           \         / 5           \ 
+Of\x  - x + 1, 3/, CRootOf\x  - x + 1, 4/]
+>>> r0, r1, r2, r3, r4 = Poly(fifth_order, x).all_roots()
+>>> r0
+        / 5           \
+CRootOf\x  - x + 1, 0/
+```
+
+Now that the roots have been found exactly, their properties can be determined
+free of numerical noise. For example, we can tell whether roots are real or not.
+If we request the {meth}`~sympy.core.expr.Expr.conjugate` (same real part and
+imaginary part with opposite sign) of a root, for example `r1`, and that is
+exactly equal to another root `r2`, that root `r2` will be returned:
+
+```py
+>>> r0.n()
+-1.16730397826142
+>>> r0.is_real
+True
+>>> r1.n()
+-0.181232444469875 - 1.08395410131771*I
+>>> r2.n()
+-0.181232444469875 + 1.08395410131771*I
+>>> r1
+        / 5           \
+CRootOf\x  - x + 1, 1/
+>>> r1.conjugate()
+        / 5           \
+CRootOf\x  - x + 1, 2/
+>>> r1.is_real
+False
+```
+
+{func}`~.solve` will also give the complex roots where possible but it is less
+efficient than using {meth}`~sympy.polys.polytools.Poly.all_roots` directly.
+
+{func}`~sympy.polys.rootoftools.RootOf` exactly represents the root in a way
+that can be manipulated symbolically, and computed to arbitrary precision. The
+{func}`~sympy.polys.rootoftools.RootOf` representation makes it possible to
+precisely:
+
+- Compute all roots of a polynomial with exact rational coefficients.
+- Decide exactly the multiplicity of every root.
+- Determine exactly whether roots are real or not.
+- Order the real and complex roots precisely.
+- Know which roots are complex conjugate pairs of each other.
+- Determine precisely which roots are rational vs irrational.
+- Represent every possible algebraic number exactly.
+
+The other numerical methods such NumPy's {external:func}`~numpy.roots`,
+{func}`~.nroots`, and {func}`~.nsolve` cannot do any of these things robustly,
+if at all. Similarly, when numerically evaluated using
+{func}`~sympy.core.evalf`, the radical expressions returned by {func}`~.solve`
+or {func}`~.roots` cannot do these things robustly.
+
+## Not All Equations Can Be Solved
+
+### Equations With No Closed-Form Solution
+
+As mentioned above, higher-order polynomials (fifth or greater) are unlikely to
+have closed-form solutions, so you may have to represent them using, for
+example, [`RootOf` as described above](#representing-roots), or use a numerical
+method such as [`nroots` as described above](#nroots).
+
+If you encounter a problem with these commands, please post the problem on the
+[mailing list](https://groups.google.com/g/sympy), or open an issue on [SymPy's
+GitHub page](https://github.com/sympy/sympy/issues). Until the issue is
+resolved, you can use another of the
```

Comment:
I don't like the wording of "If you encounter a problem" here. I am increasingly finding that I have to close GitHub issues that are really just support requests. Anything suggesting to open a GitHub issue should clearly distinguish between asking for help (which should not be done on GitHub) and reporting a bug. Pointing to the mailing list is fine but not to GitHub.

Reply:
I changed the term from "problem" to "bug", though users may not be able to distinguish between a problem with their usage of commands vs. a bug in the command itself. Please let me know if that does not fully address your comment--given that change, do you think we should remove the link to the GitHub issues page?

Reply:
Many users will struggle to tell the difference between what is or isn't a bug. I would like those things have some level of triage outside of GitHub though e.g. if they ask on the mailing list then someone can say if it is a bug and if it should be reported to GitHub.

The problem with GitHub issues is that they have to be "open" or "closed" and our workflow needs to close them as soon as possible but that can easily come across as rude e.g. #24196. Managing the list of over 4000 open issues is a lot of work so we need to be very careful not to encourage people to use them incorrectly.

I would rather not refer to GitHub at all on this page which is directed at beginners.

Reply:
Understand. Earlier in the project, @asmeurer endorsed including links to the listserv and GitHub issues page, so I just want to get his input on removing the GitHub link.

The same link is on the other solving sub-pages, so if I remove it here, I'd presumably remove it on all of the other pages too.

Reply:
Personally I don't really have a problem with linking the issue page. Issues that are not really issues can be converted into discussions, which don't have the open/closed problem.

Reply:
Just figuring out whether an issue is an issue is work. The discussions feature is a terrible idea because it mixes up things that are not issues into the GitHub workflow when they really belong elsewhere. Converting issues into discussions is not a general solution because it isn't possible to revert that and turn the discussion back into an issue.

The real problem with the discussion feature is that it is only seen by the people who subscribe to GitHub notations. That means that any support request is placing drain on precisely the same people who would have seen it if it was an issue. The point about directing to the mailing list is that there are lots of *other* people who can potentially help (or ignore if they choose).

Note importantly though that what I initially objected to is the wording "If you encounter a problem". That is not a good criteria for opening a GitHub issue. Too many people already get confused about that so we should not put a note in the docs that encourages precisely the wrong understanding of what GitHub issues are for.

Right now people open issues and it is not even clear whether *they* think that it is an issue or if it is just a support request. We should make be very clear that GitHub issues are not for support requests.

Take a look through the recently closed issues to get a sense for what is happening:
https://github.com/sympy/sympy/issues?q=is%3Aissue+is%3Aclosed

Reply:
Case in point: #24220. I'll let someone else respond. Most of these I deal with myself but I'm losing patience and if you want to direct more people my way then I'll just give up and leave it to others.

Reply:
@asmeurer and I decided to remove the links to the GitHub issues page for all the solving guide pages, by creating a pull request after the pull requests for all the new pages are merged. I'll also rename "Report a problem" to "Report a bug" for all the pages in that PR.

Reply:
We should be able to alleviate this problem using the issue templates feature. For example, look at what the [NumPy](https://github.com/numpy/numpy/issues/new/choose) new issue page looks like, with a link for a normal issue report page and a link to the mailing list for support requests.